src/atoms/CarouselSlot.js

```diff
@@ -6,30 +6,20 @@ import { getVariable } from '../utils/get-variable';
 
 export const CarouselSlot = styled(Col)`
 	${(props) => {
-		const base = css`
+		const horizontalBase = getVariable('horizontalBase')(props);
+		const verticalBase = getVariable('verticalBase')(props);
+
+		const marginRight = (!props.vertical) ? `calc(1/2 * ${horizontalBase})` : '0';
```

Comment:
I'm wondering if Col has default margin-right and margin-bottom of 0?

Reply:
It should, yes. How do you envision the override to be done?

Reply:
if so, then it's fine as it is!